governance/polls/Activate SparkLend Freezer Mom - January 8, 2024.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Activate SparkLend Freezer Mom - January 08, 2024
+summary: Signal your support or opposition to activating the SparkLend Freezer Mom contract
+discussion_link: https://forum.makerdao.com/t/spark-spell-proposed-changes/23298
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-01-08T16:00:00
+end_date: 2024-01-11T16:00:00
+---
+# Poll: Activate SparkLend Freezer Mom - January 8, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday,January 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the activation of the SparkLend Freezer Mom contract on Ethereum. The SparkLend Freezer Mom contract allows governance to bypass the 2-day GSM window to freeze or pause any SparkLend markets.
```

Comment:
```suggestion
The community can vote in this poll to express support or opposition to the activation of the SparkLend Freezer Mom contract on Ethereum. The SparkLend Freezer Mom contract allows governance to bypass the 2-day GSM window to freeze or pause any SparkLend markets but also to reverse these actions when deemed necessary.
```

Reply:
In addition I would also explain the rationale of this change. Explaining that the addition of these functionalities ensures that Maker Governance has the flexibility to react swiftly in emergencies. But it is good as is.